v4/export/ir_impl_test.go

```diff
@@ -10,15 +12,25 @@ var _ = Suite(&testIRImplSuite{})
 type testIRImplSuite struct{}
 
 type simpleRowReceiver struct {
-	data string
+	data []string
 }
 
-func (s *simpleRowReceiver) BindAddress(arg []interface{}) {
-	arg[0] = &s.data
+func newSimpleRowReceiver(length int) *simpleRowReceiver {
+	return &simpleRowReceiver{data: make([]string, length)}
+}
+
+func (s *simpleRowReceiver) BindAddress(args []interface{}) {
+	for i := range args {
+		args[i] = &s.data[i]
+	}
 }
 
 func (s *simpleRowReceiver) ReportSize() uint64 {
-	panic("not implement")
+	sum := 0
```

Comment:
The type of `sum` should be `uint64` and the cast to uint64 should happen on line 31